packetbeat/flows/worker_test.go

```diff
@@ -116,7 +116,7 @@ func TestCreateEvent(t *testing.T) {
 
 	// Write the event to disk if -data is used.
 	if *dataFlag {
-		event.Fields.Put("@timestamp", common.Time(end)) //nolint:errcheck // Never fails.
```

Comment:
why was this removed?

Reply:
It was failing the lint check saying this is not used.